docker/Dockerfile

```diff
@@ -1,6 +1,5 @@
 FROM --platform=$BUILDPLATFORM golang:1.18 as builder
-RUN apt-get update && \
-    apt-get install make bash
```

Comment:
Why do we remove 'update' and bash installation?

Reply:
Because this is the builder image and these aren't needed to perform the build successfully. Just hygiene.